lib/src/components/form/form.dart

```diff
@@ -1349,12 +1349,14 @@ class FormField<T> extends StatelessWidget {
   final Widget label;
   final Widget? hint;
   final Widget child;
+  final Widget? secondaryChild;
```

Comment:
Honestly great addition, but can we call this `trailing` instead? Also `trailing` wouldn't be complete without `leading` widget (placed before the label)

Reply:
I will add the label and rename it to trailing